packages/jsxstyle-utils/src/componentStyles.ts

```diff
@@ -1,3 +1,7 @@
+export interface StyleObject {
+  [key: string]: { [key: string]: string | number } | null;
```

Comment:
For our project, I lifted [CSS types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L817) from `@types/react`. It's pretty important for `jsxstyle` to have strict types for CSS props. Otherwise any prop on any component goes, which would hide a ton of errors.